### PR TITLE
style buttons

### PR DESCRIPTION
**Pull request by kesiah**
Test URLs:
- Before: https://main--vg-macktrucks-com-rd--netcentric.hlx.page/drafts/laura/
- After: https://buttons--vg-macktrucks-com-rd--netcentric.hlx.page/drafts/laura/

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
...
+  transition: all var(--duration-small) var(--easing-standard);
+}
+
+.redesign-v2 a.button.primary.small,
+.redesign-v2 button.primary.small,
+.redesign-v2 a.button.secondary.small,
+.redesign-v2 button.secondary.small,
+.redesign-v2 a.button.tertiary.small,
+.redesign-v2 button.tertiary.small {
+  padding: 10px 20px 6px;
```
Comment (cogniSyb) [inline styles/styles.css:850]: Too many classes in use here. Why no BEM? As all the small/medium/large buttons have the same sizing, I expect no selectors in use for primary/secondary/tertiary buttons

File: styles/styles.css [hunk 2]
```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
...
+  font-weight: normal;
+  gap: 10px;
+  justify-content: center;
+  letter-spacing: 1.12px;
+  line-height: var(--button-line-height);
+  min-width: 80px;
+  padding: 13px 20px 9px;
+  text-align: center;
+  text-decoration: none;
+  transition: all var(--duration-small) var(--easing-standard);
```
Comment (cogniSyb) [inline styles/styles.css:841]: Transitions shouldn’t have `all`. Please replace with `background-color`

File: styles/styles.css [hunk 3]
```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
...
 }
 
+/* REDESIGN Buttons */
+.redesign-v2 a.button.primary,
+.redesign-v2 button.primary,
+.redesign-v2 a.button.secondary,
+.redesign-v2 button.secondary,
+.redesign-v2 a.button.tertiary,
+.redesign-v2 button.tertiary {
+  align-items: center;
```
Comment (cogniSyb) [inline styles/styles.css:826]: Can’t we add a button class to a button? I think these selectors can be way cleaner

File: styles/styles.css [hunk 4]
```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
...
+  display: inline-flex;
+  font-family: var(--ff-body);
+  font-size: var(--button-font-size);
+  font-weight: normal;
+  gap: 10px;
+  justify-content: center;
+  letter-spacing: 1.12px;
+  line-height: var(--button-line-height);
+  min-width: 80px;
+  padding: 13px 20px 9px;
```
Comment (cogniSyb) [inline styles/styles.css:838]: The button sizing is off. I see 96×32px for small, 128×40px for medium, and 160×48px for large buttons.

File: styles/styles.css [hunk 5]
```diff
@@ -808,6 +816,100 @@ main .section.responsive-title h1 {
...
+}
+
+.redesign-v2 a.button.primary,
+.redesign-v2 button.primary {
+  background-color: var(--button-primary-red-enabled);
+  color: var(--c-primary-white);
+}
+
+.redesign-v2 a.button.primary:hover,
+.redesign-v2 button.primary:hover {
```
Comment (cogniSyb) [inline styles/styles.css:869]: Add for all :hover selectors :focus as well

**Review by cogniSyb — CHANGES_REQUESTED** [review]

**Comment by cogniSyb** [comment]
Looking at the overall designs of Mack, I think it’s good to rename primary to CTA. Secondary (gold) would become primary Then the code + usage of button is more in line with Volvo.

File: styles/styles.css (hunk 2 above)
Comment (kesiah) [reply styles/styles.css]: applying to color, border-color and background-color now

**Review by Lakshmishri — APPROVED** [review]

**Review by TomaszDziezykNetcentric — APPROVED** [review]

**Pull request merged**